content/docs/contributing/to-docs/get-started.md

```diff
@@ -1,108 +1,231 @@
-
-   5. The PR contains the appropriate *Labels*, if required. Labels make it easier to sort through lists of PRs and may specify whether a PR is a work in progress, seeking feedback and review, that should not be merged into `o3de.org/main`. For more information, see [Submit a PR](./submit-a-pr).
-
-1. **Respond to PR feedback:** Feedback will come in the form of comments and *suggestions* that can be committed from the GitHub PR interface. It's important to understand that the PR process is a collaborative discussion. Every comment does not need to be addressed, and every suggestion does not need to be integrated. When the contributor and two reviewers approve of the contribution, it can be integrated. Here are a few tips for addressing suggestions and comments:
-
-   1. To commit multiple suggestions, use the batch functionality to integrate multiple suggestions in a single commit.
-
-   2. When addressing comments, maintain the relative conversation in the PR, edit the topics as necessary, and commit the changes to your fork/branch. Your new commits will be automatically added to the PR.
-
-   3. Make sure to request a re-review of your new commits if required. 
-
-1. **The PR is merged and closed:** When the comments and suggestions have been addressed, and two reviewers have approved the PR, it can be merged into `o3de.org/main`. **Never** merge your own PRs, the final reviewer to sign off is responsible for the merge. The PR is automatically closed when merged.
-
-1. **The related issue is closed:** If you have included a related issue number in the title of your PR, the related issue will be automatically closed when the PR is closed. Your contribution is complete. Congrats!
+---
+linktitle: Get Started
+title: Get Started Contributing to Docs
+description: Get started contributing to Open 3D Engine (O3DE) docs by setting up your local repo, setting up your writing environment, and submitting docs. 
+toc: true
+weight: 200
+---
+
+This section will help you get started on contributing to Open 3D Engine (O3DE) documentation. You'll learn how to set up your writing environment and about the processes you need to get your documentation from text editor to the official O3DE documentation. 
+
+O3DE documentation uses the [fork and pull model](https://en.wikipedia.org/wiki/Fork_and_pull_model) for contributions. As a contributor, you maintain a fork (your own repository of the O3DE documentation) on GitHub, and work locally to edit documentation. You then submit PRs from your fork for review.
+
+{{< note >}}
+The steps on this page use the terminal to run Git commands. You may complete these steps through an alternative tool of your choice, but ensure that you're completing the correct operations. 
```

Comment:
nit: Alternative is most likely the GitHub UX, do you want to link to their instructions?

Reply:
Adding information on this